static/app/components/modals/debugFileCustomRepository/appStoreConnect/index.tsx

```diff
@@ -361,10 +357,11 @@ function AppStoreConnect({
           data: {sessionContext},
         }
       );
-
+      setUseSms(true);
       setSessionContext(response.sessionContext);
       addSuccessMessage(t("We've sent a SMS code to your phone"));
     } catch {
+      // TODO: use the message from the itunes-sms-blocked-error
```

Comment:
@priscilawebdev It it possible to change the message based on the response here?  We return a special "blocked" response from the backend with `code = itunes-sms-blocked-error` and a message: https://github.com/getsentry/sentry/blob/9423b9d13fe68636db0287b20c20b9d5e20d9fb8/src/sentry/api/exceptions.py#L118-L121. It would be really helpful to show this to the user because in that case "please try again" is the wrong advice and will only make things worse.

Reply:
sure it is possible but I would have anyway to use convert it to translations on the client side

Reply:
https://github.com/getsentry/sentry/blob/master/static/app/views/organizationGroupDetails/grouping/errorMessage.tsx